automated_analysis.py

```diff
@@ -55,26 +55,20 @@
 
     messages_json_input_path = args.messages_json_input_path
     individuals_json_input_path = args.individuals_json_input_path
-    output_dir = args.output_dir
```

Comment:
Rem to update docker-run-automated-analysis.sh to refer to AUTOMATED_ANALYSIS_OUTPUT_DIR

Reply:
Updated, and changed output-graphs/ to automated-analysis-outputs/ too, thanks